README.md

```diff
@@ -68,4 +68,33 @@ module "mybastion" {
 }
 ```
 
-3) Merge the change and a new bastion will be deployed.
\ No newline at end of file
+3) Merge the change and a new bastion will be deployed.
+
+### Updating image used by bastion vm
+
+The image used by the bastion vm is stored in a shared image gallery called hmcts. The image is generated by the Azure pipeline in hmcts/bastion-packer.
+
+Whenever a new version of the image is needed, e.g. a new software package needs installed, do the following:
+
+1. Update the provision-bastion bash script with the changes needed
+2. Update the ubuntu.pkr.hcl file with a new image version
+
+Upon the pipeline running successfully, you should see a new image version in the shared image gallery.
+
+Once you've done the above, update the 10-locals.tf file for each environment to point to the new image version denoted by image_id:
+
+```dtd
+locals {
+  environment             = "sbox"
+  location                = "uksouth"
+  image_id                = "1.0.2"
```

Comment:
this should be called `image_version` not id

Reply:
Updated